src/components/AppWrapper.spec.jsx

```diff
@@ -1,7 +1,12 @@
 import 'isomorphic-fetch'
 import { setupAppContext } from './AppWrapper'
 
-jest.mock('cozy-client')
+jest.mock('cozy-client', () => ({
+  __esModule: true,
+  default: {
+    fromDOM: jest.fn(() => ({ registerPlugin: jest.fn() }))
+  }
+}))
```

Comment:
I think it would be preferable to only mock fromDOM instead of mocking all the cozy-client module, and that fromDOM return a real cozy-client. This way, you only bypass a small code path, and the tests would be closer to what is done for real.

Reply:
Merci pour l'astuce effectivement.
On va revenir à un classique `new CozyClient` après le merge de https://github.com/cozy/cozy-client/pull/968